--- ./configure.in.orig	Mon Feb 11 16:18:24 2002
+++ ./configure.in	Sat Aug 17 12:56:20 2002
@@ -42,17 +42,14 @@
 nlsdir='${prefix}/share/locale'
 AC_SUBST(nlsdir)
 
-AC_OUTPUT([
-gtkballs.lsm
-gtkballs.spec
-Makefile
-src/Makefile
-doc/Makefile
-doc/gtkballs.6x
-intl/Makefile
-gtkballs-data/Makefile
-intl/Makefile
-po/Makefile.in],
-[touch Makefile.in
-sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile]
-)
+AC_OUTPUT([gtkballs.lsm
+	   gtkballs.spec
+	   Makefile
+	   src/Makefile
+	   doc/Makefile
+	   doc/gtkballs.6x
+	   gtkballs-data/Makefile
+	   intl/Makefile
+	   po/Makefile.in],
+	   [touch Makefile.in
+	    sed -e "/POTFILES =/r po/POTFILES" po/Makefile.in > po/Makefile])
