--- ./po/Makefile.in.in.org	Mon Oct  1 19:40:31 2001
+++ ./po/Makefile.in.in	Sat Aug 17 12:53:21 2002
@@ -111,9 +111,9 @@
 install-data-no: all
 install-data-yes: all
 	if test -r "$(MKINSTALLDIRS)"; then \
-	  $(MKINSTALLDIRS) $(datadir); \
+	  $(MKINSTALLDIRS) $(DESTDIR)$(datadir); \
 	else \
-	  $(SHELL) $(top_srcdir)/mkinstalldirs $(datadir); \
+	  $(SHELL) $(top_srcdir)/mkinstalldirs $(DESTDIR)$(datadir); \
 	fi
 	@catalogs='$(CATALOGS)'; \
 	for cat in $$catalogs; do \
--- ./gtkballs-data/Makefile.am.org	Tue Nov 20 11:39:37 2001
+++ ./gtkballs-data/Makefile.am	Sat Aug 17 12:51:29 2002
@@ -1,16 +1,16 @@
 EXTRA_DIST=gtkballs-logo.xpm gtkballs_16x16.xpm themes
 
 install-data-local:
-	$(mkinstalldirs) $(datadir)/gtkballs
-	$(INSTALL_DATA) *.xpm $(datadir)/gtkballs
-	$(mkinstalldirs) $(datadir)/gtkballs/themes
+	$(mkinstalldirs) $(DESTDIR)$(datadir)/gtkballs
+	$(INSTALL_DATA) *.xpm $(DESTDIR)$(datadir)/gtkballs
+	$(mkinstalldirs) $(DESTDIR)$(datadir)/gtkballs/themes
 	for d in themes/*; do \
-	    $(mkinstalldirs) $(datadir)/gtkballs/$$d; \
+	    $(mkinstalldirs) $(DESTDIR)$(datadir)/gtkballs/$$d; \
 	    for i in $$d/*; do \
-		$(INSTALL_DATA) $$i $(datadir)/gtkballs/$$d; \
+		$(INSTALL_DATA) $$i $(DESTDIR)$(datadir)/gtkballs/$$d; \
 	    done; \
 	done
-	$(mkinstalldirs) $(localstatedir)
+	$(mkinstalldirs) $(DESTDIR)$(localstatedir)
 
 uninstall-local:
 	rm -rf $(datadir)/gtkballs
--- ./Makefile.am.org	Fri Feb  8 17:07:15 2002
+++ ./Makefile.am	Sat Aug 17 12:50:15 2002
@@ -11,8 +11,8 @@
 	     gtkballs.wmconfig
 
 install-data-local:
-	$(mkinstalldirs) $(datadir)/gtkballs
-	$(INSTALL_DATA) COPYING $(datadir)/gtkballs
+	$(mkinstalldirs) $(DESTDIR)$(datadir)/gtkballs
+	$(INSTALL_DATA) COPYING $(DESTDIR)$(datadir)/gtkballs
 
 install-data-hook:
-	if [ ! -f $(localstatedir)/gtkballs-scores ]; then touch $(localstatedir)/gtkballs-scores; chmod 0666 $(localstatedir)/gtkballs-scores; fi
\ No newline at end of file
+	if [ ! -f $(DESTDIR)$(localstatedir)/gtkballs-scores ]; then touch $(DESTDIR)$(localstatedir)/gtkballs-scores; chmod 0666 $(DESTDIR)$(localstatedir)/gtkballs-scores; fi
